scripts/ci/run_perftest.sh

```diff
@@ -33,9 +33,7 @@ cat << EOF > /etc/telegraf/telegraf.conf
   json_time_key = "time"
   json_time_format = "unix"
   tag_keys = [
-    "i_type",
-    "branch",
-    "commit"
+    "i_type"
```

Comment:
Checking my understanding: `branch` and `commit` turn into fields by removing them from this list, because they're already included in `json_string_fields` above?

Reply:
I believe that's correct, yes. Although tbh I don't have a real firm grasp on the semantics of difference between field and tag. But originally these items were not included in the list of tags and so appeared differently in queries and cloud2 displays